loopchain/baseservice/peer_object.py

```diff
@@ -138,13 +138,14 @@ def __init__(self, channel: str, peer_info: PeerInfo):
         self.__no_response_count = 0
         self.__channel = channel
 
-        self.__create_live_data()
+        self.__create_live_data(p2p_service)
 
-    def __create_live_data(self):
+    def __create_live_data(self, p2p_service):
         try:
-            self.__stub_manager = StubManager(self.__peer_info.target,
-                                              loopchain_pb2_grpc.PeerServiceStub,
-                                              conf.GRPC_SSL_TYPE)
+            target = self.__peer_info.target
+            p2p_service.add_client(target)
```

Comment:
`add_client` seems unclear. It means adding `target`. But `get_client` does not return `target`. Even `target` is passed as a parameter.  So `add_client` does not add target as a client.

How about separating it?
```python
client = p2p_service.create_client(target)
p2p_service.add_client(target, client)
```

Reply:
This `peer_object` remove.
I will consider next phase. thank you.